docs/guide/security_checklist.md

```diff
@@ -0,0 +1,118 @@
+# Security checklist
+
+When getting ready to go live with your project for the first time, or when re-launching it,
+you need to make sure your setup is secure.
+
+!!! caution
+
+    Security is an ongoing process. After going live, you should pay attention to security advisories
+    released via [your service portal,](https://support.ibexa.co/)
+    or via [Security advisories](https://developers.ibexa.co/security-advisories) if you're not a subscriber.
+
+## Symfony
+
+### `APP_SECRET`
+
+`APP_SECRET` needs to be a strong, random, securely stored value.
+
+- Do not use a default value like `ff6dc61a329dc96652bb092ec58981f7` or `ThisTokenIsNotSoSecretChangeIt`
+- The secret must be secured against unwanted access. Do not commit it to version control.
+- The secret must be long enough. 32 characters is minimum, longer is better.
+
+!!! tip
+    
+    This will generate a 64-character-long secure random secret:
+    
+    `print bin2hex(random_bytes(32));`
+    
+!!! tip
+
+    On Ibexa Cloud, if `APP_SECRET` is not set, the system sets it to [`PLATFORM_PROJECT_ENTROPY`](https://docs.platform.sh/development/variables.html#platformsh-provided-variables)
+
+### Symfony production mode
+
+Only expose Symfony production mode openly on the internet. If you have exposed dev mode, secure and change all passwords/secrets.
+Exposing the dev mode exposes things like `phpinfo`, environment variables and so on.
+
+!!! tip "More about Symfony security"
+
+    - [Authentication and authorisation](https://symfony.com/doc/current/security.html), and [more on this subject](https://symfony.com/doc/current/security.html#learn-more)
+    - Symfony's [secrets management system](https://symfony.com/doc/current/configuration/secrets.html)
+
+## [[= product_name_ee =]]
+
+### Fully-vetted admin users
+
+Make sure Admin users and other privileged users who have access to System Information and setup in the back end
+are vetted and fully trustworthy.
+
+As administrator you have access to full information about the system through the `setup/system_info` Policy,
+and also to user data, Role editing, and many other critical aspects.
+
+### Strong passwords
+
+Enforce strong passwords for all users. This is specially important for admin accounts and other privileged users.
+
+- Never go online with admin password set to `publish` or any other default value
+- Introduce password quality checks. Make sure they are secure enough (length/complexity).
```

Comment:
```suggestion
- Introduce password quality checks. Make sure passwords are secure enough (length/complexity).
```

Reply:
@dabrt Here "they" refers to the quality checks.
```suggestion
- Introduce password quality checks. Make sure the checks are strict enough (length/complexity).
```